WPThemeReview/Tests/CoreFunctionality/PrefixAllGlobalsUnitTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Unit test class for WPThemeReview Coding Standard.
+ *
+ * @package WPTRT\WPThemeReview
+ * @link    https://github.com/WPTRT/WPThemeReview
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WPThemeReview\Tests\CoreFunctionality;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+/**
+ * Unit test class for the PrefixAllGlobals sniff.
+ *
+ * @package WPTRT\WPThemeReview
+ *
+ * @since 0.2.0
+ */
+class PrefixAllGlobalsUnitTest extends AbstractSniffUnitTest {
+
+	/**
+	 * Error files with the expected nr of errors.
+	 *
+	 * @var array
+	 */
+	private $expected_results = array(
+
+		/*
+		 * In /CoreFunctionality/ThemeTemplatesException.
+		 */
+
+		// Files in a theme context.
+		'footer_widgets.inc' => 1,
+		'header.inc'         => 1,
```

Comment:
You should probably also add some test files using the `allowed_folders` without a typical template name, i.e. `/partials/something.php`.

And a test file which _should_ still trigger the error, i.e. a file with a non-typical template name, like `something.php`.